src/validate.ts

```diff
@@ -32,14 +37,21 @@ interface ValidationResult {
   messages: ConventionalCommitMessage[];
 }
 
-function outputErrors(
+/**
+/* Takes a single non-compliant commit `message`, its `sha`, and its list of
+ * `errors`, and outputs them to `core.error` if the parameter `useErrorLevel`
+ * is `true`, or `core.warning` otherwise.
+ * If `sha` is `undefined`, the message is assumed to be a pull request title
+ * and the output will reflect that.
+ */
+function outputCommitErrors(
   message: string,
   errors: LlvmError[],
-  sha: string | undefined
+  sha: string | undefined,
```

Comment:
```suggestion
  sha?: string,
```

Reply:
That's not an option here, as I don't want the parameter to be optional.